src/tests/stub_libip.cc

```diff
@@ -12,34 +12,122 @@
 #define STUB_API "ip/libip.la"
 #include "tests/STUB.h"
 
+#include "ip/Address.h"
+namespace Ip
```

Comment:
Here and elsewhere in this PR, please define stubbed globals and class members the same way we define regular globals and members:

```C++
void Namespace::Class::method() STUB
bool Namespace::Class::StaticMember = false;
int Namespace::Global = 0;
```

rather than 

```C++
namespace Namespace
{
void Class::method() STUB
bool Class::StaticMember = false;
int Global = 0;
}
```

Our regular style is slightly safer when it comes to globals, and style consistency helps searching and adjusting code.

Reply:
Done in[2323d3c](https://github.com/squid-cache/squid/pull/1254/commits/2323d3c17f7de8e45b28fd13a20e315fc11e9ea8)